config/routes.rb

```diff
@@ -0,0 +1,3 @@
+UiComponents::Engine.routes.draw do
```

Comment:
So you are supposed to mount the styleguide in the rails app that uses ui_c? Is there a nice way to do so? What about security?